interchaintest/liquidstake_stkxprt_test.go

```diff
@@ -0,0 +1,272 @@
+package interchaintest
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"cosmossdk.io/math"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/strangelove-ventures/interchaintest/v7/testutil"
+	"github.com/stretchr/testify/require"
+
+	"github.com/persistenceOne/persistenceCore/v11/interchaintest/helpers"
+	liquidstaketypes "github.com/persistenceOne/pstake-native/v2/x/liquidstake/types"
+)
+
+// TestLiquidStakeStkXPRT runs the flow of liquid XPRT staking using
+// liquidstake module, including LSM-LP flow when stake gets locked into Superfluid LP.
+func TestLiquidStakeStkXPRT(t *testing.T) {
```

Comment:
@puneet2019 you want to look at this flow and raise any concerns

Reply:
looks good